frontend-nextjs/features/(dashboard)/FileUpload.tsx

```
@@ -112,8 +114,13 @@
 		const data = await response.json();
 	  let success_upload_signedurl = true;
 		for (const file of files) {
-			//NFC正規化
-			file.name = file.name.normalize("NFC");
+			let org_file_name = file.name;
+			if (user) {
+				file.name = user.uid + "/" + file.name;
+				file.name = file.name.normalize("NFC");
+			} else {
+				throw new Error("User is not authenticated");
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)